transforms/sort.go

```diff
@@ -43,3 +44,22 @@ func Sort(less func(i, j optimus.Row) (bool, error)) optimus.TransformFunc {
 		return nil
 	}
 }
+
+// StableSort takes in a function that reports whether the row i should sort before row j.
+// It outputs the rows in stably sorted order.
+func StableSort(less func(i, j optimus.Row) (bool, error)) optimus.TransformFunc {
```

Comment:
I suspect you'll resist, but I'd be in favor of factoring out the common logic of this and the previous function into a helper that accepts a lambda.  Then `Sort` and `StableSort` would each be one line: `return sort(less, sort.Sort)` and `return sort(less, sort.Stable)`.